### DISCUSSION TITLE
How to change border color for active pane ?

### DISCUSSION THREAD

**Discussion by lcheylus** [wezterm/wezterm#3337] [Q&A] [answered]
Hi,

I'm using Wezterm with multiple panes and no tab like I'm used to with Kitty terminal or tmux multiplexer.

I would like to change the color of borders for the active pane. I know it's possible to style Inactive Panes (see https://wezfurlong.org/wezterm/config/appearance.html?h=inactive#styling-inactive-panes) for HSB. But is it possible to change color scheme for Active Pane ?

**Comment by nekowinston** [comment, marked as answer]
AFAIK, the colorscheme would be set via [`set_config_overrides`](https://wezfurlong.org/wezterm/config/lua/window/set_config_overrides.html) which happens on the `window`, so I don't think you can do it per-pane.

**Reply by wez** [reply]
Correct: overriding config at the pane level is not supported.
There also is not a way to change the pane border color based on which pane is active today either.

**Reply by juru1234** [reply]
Would be really nice to have the pane border color based on the active pane. I really don't like the dimming stuff when you are in a terminal but still want to read code in another pane...

**Comment by lcheylus** [comment]
Thanks for the explanations.

**Comment by GZLiew** [comment]
is there no way to around this?

**Reply by mozumasu** [reply]
As a workaround, you can make it easier to distinguish active and inactive panes by adjusting their colors, for example using inactive_pane_hsb as shown below.

The upper-right pane is active.

```lua:~/.config/wezterm/wezterm.lua
---@type Wezterm
local wezterm = require("wezterm")
local config = wezterm.config_builder()

config.inactive_pane_hsb = {
  hue = 0.9, -- Slightly shift hue to reduce the blue tint
  saturation = 0.9,
  brightness = 1.0,
}
config.colors = {
  -- A bluish background color (appears blue in the active pane)
  background = "#1a1a2e",
}
```

<img width="1918" height="1048" alt="スクリーンショット 2026-01-03 16 51 50" src="https://github.com/user-attachments/assets/f988bf18-c6fd-4413-adde-516a402cdad7" />

**Reply by mmercurio** [reply]
Is there a way to get something like `config.inactive_pane_hsb` to work at the window level? 

When using `config.inactive_pane_hsb` as-is, it only identifies active/inactive panes within a window. If I have multiple WezTerm windows open, it would be nice to see visual differences between the active window that has focus vs other windows that don't.
